docs/usage.md

```diff
@@ -143,28 +255,33 @@ process.$multiqc.module = []
 
 ### `--mature`
```

Comment:
Need to add all of these parameters to TOC at top of file.

Reply:
done.